.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,158 @@
+        run: ./scripts/load_templates.sh
+      - name: Compile translations
+        run: make translate
+      - name: Run travis steps
+        run: ln -sf .development.env .env
+      - name: Running translation tests
+        run: pipenv run python -m scripts.extract_translation_templates --test
+      - name: Running lint tests
+        run: pipenv run ./scripts/run_lint_python.sh
+      - name: Running unit tests
+        run: pipenv run ./scripts/run_tests_unit.sh
+  validate-schemas:
+      runs-on: ubuntu-latest
+      services:
+        validator:
+          image: onsdigital/eq-schema-validator:v3
+          ports:
+            - 5001:5000
+      steps:
+        - uses: actions/checkout@v2
+        - name: Running schema tests
+          run: ./scripts/validate_test_schemas.sh
+  test-functional:
+      needs: [python-dependencies, node-dependencies]
+      runs-on: ubuntu-latest
+      env:
+        EQ_RUN_FUNCTIONAL_TESTS_HEADLESS: True
+      steps:
+        - uses: actions/checkout@v2
+        - uses: actions/setup-node@v1
+          with:
+            node-version: '8.16.0'
+        - run: |
+            echo "::set-env name=PYTHON_VERSION::$(cat .python-version)"
+        - uses: actions/setup-python@v1
+          with:
+            python-version: ${{ env.PYTHON_VERSION }}
+        - name: Install pipenv
+          run: pip install pipenv==2018.11.26
+        - name: Cache virtualenv
+          id: cache-virtualenv
+          uses: actions/cache@v1
+          with:
+            path: ~/.local/share/virtualenvs/
+            key: ${{ runner.os }}-${{ env.PYTHON_VERSION }}-virtualenvs-${{ hashFiles('Pipfile.lock') }}
+        - name: Get yarn cache
+          id: get-yarn-cache
+          run: echo "::set-output name=dir::$(yarn cache dir)"
+        - name: Cache yarn modules
+          uses: actions/cache@v1
+          id: cache-yarn
+          with:
+            path: ${{ steps.get-yarn-cache.outputs.dir }}
+            key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
+        - name: Install yarn deps
+          run: yarn install
+        - name: Functional tests spec lint
+          run: ./scripts/lint_functional_test_specs.sh
+        - name: Javascript linting check
+          run: yarn lint
+        - name: Docker compose
+          run: docker-compose --version && docker-compose up --build -d
+        - name: Functional tests
+          run: ./scripts/run_tests_functional.sh
+        - name: Docker compose shutdown
+          run: docker-compose kill
+  docker-push:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v2
+        - name: Write app version
+          run: printf $(git rev-parse HEAD) > .application-version
+        - name: Tag
+          run: |
+            export BRANCH=$(git rev-parse --abbrev-ref HEAD)
+            export TAG=`if [ "$BRANCH" == "" ]; then echo "latest"; else echo branch-$BRANCH; fi`
+            echo "::set-env name=TAG::$TAG"
```

Comment:
What does this do?

Reply:
Sets a tag environment variable to be used in another step. Exporting it won't pass it to the next step. https://help.github.com/en/actions/automating-your-workflow-with-github-actions/development-tools-for-github-actions#set-an-environment-variable-set-env

Reply:
Sets a tag environment variable to be used in another step. Exporting it won't pass it to the next step. https://help.github.com/en/actions/automating-your-workflow-with-github-actions/development-tools-for-github-actions#set-an-environment-variable-set-env

Reply:
Ah cool. 👍